src/pykeen/triples/triples_factory.py

```diff
@@ -745,62 +895,76 @@ def _tf_cleanup_all(
         else:
             reference, other = _tf_cleanup_deterministic(reference, other)
         rv.append(other)
-    return [reference, *rv]
+    return reference, *rv
```

Comment:
this fails in py37, so you have to keep the brackets

Reply:
5215a14